repos/fdbt-site/src/pages/selectPurchaseMethods.tsx

```diff
@@ -286,11 +286,13 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
                   )
                   .reduce((result, item) => ({ ...result, [item[0]]: item[1] }), {}));
 
+    const isCapped = fareType === 'capped';
+
```

Comment:
![Screenshot 2022-12-19 at 15 15 03](https://user-images.githubusercontent.com/111580926/208458217-292f6d6c-0cd2-4553-b6a4-5c9fc0c66254.png)

Reply:
The text needs to say 'You currently have no saved capped purchase methods'. And the text above needs to say 'You can create new capped purchase methods in your.... etc'